pkg/functions/runner_test.go

```diff
@@ -1,62 +1,44 @@
 //go:build !integration
 // +build !integration
 
-package functions_test
+package functions
 
 import (
 	"context"
-	"fmt"
-	"net/http"
+	"errors"
 	"testing"
-
-	fn "knative.dev/func/pkg/functions"
-	"knative.dev/func/pkg/oci"
-	. "knative.dev/func/pkg/testing"
 )
 
-// TestRunner ensures that the default internal runner correctly executes
-// a scaffolded function.
-func TestRunner(t *testing.T) {
```

Comment:
This test was moved to the client test such that this file could be brought inside the functions package, and thereby allowing the TestGetRunFuncErrors test to be writen which unit-tests the private getRunFunc function.